apps/easypid/src/features/share/FunkeRequestedAttributesDetailScreen.tsx

```diff
@@ -47,41 +75,90 @@ export function FunkeRequestedAttributesDetailScreen({
     return null
   }
 
+  const displayedMetadata = useMemo(() => {
```

Comment:
can't we just not render undefined fields? this feels a bit redundant

Reply:
Hmm the formatDate is needed though

Reply:
Shouldn't we do that in the getCredentialForDisplay?

Reply:
👍